index.html

```diff
@@ -10630,8 +10630,19 @@ <h2 id="AudioWorklet-Examples">
             <dd>
               <p>
                 This method is DEPRECATED. It is equivalent to setting
-                <code>orientationX</code>, <code>orientationY</code>, and
-                <code>orientationZ</code> AudioParams directly.
+                <code>orientationX.value</code>,
+                <code>orientationY.value</code>, and
+                <code>orientationZ.value</code> attribute directly, with the
```

Comment:
Should orientationX,Y,Z be linkified? Just asking; I'm ok if you don't do this.

Reply:
I've done it.